ci/container-canary/base.yml

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
```

Comment:
I wonder if it would be more clear to put these files in `tests/` to more clearly show what they are.

Reply:
sure! Just did that in https://github.com/rapidsai/docker/pull/670/commits/4d6452885a032ae8b033ddbb77b7eee8e5a7f8e1